core/README.md

```diff
@@ -25,29 +25,42 @@ version of embedded-graphics-core may be used for multiple major versions of emb
 
 ### Core functionality
 
-* `DrawTarget` - By implementing a draw target for a display driver, all embedded-graphics drawables can be drawn to that display.
-* `Drawable` - This trait can be implemented to make an object drawable to any `DrawTarget`. Examples include shapes, text, UI elements, etc.
-* `ImageDrawable`
+* [`DrawTarget`] - By implementing a draw target for a display driver, all embedded-graphics drawables can be drawn to that display.
+* [`Drawable`] - This trait can be implemented to make an object drawable to any [`DrawTarget`]. Examples include shapes, text, UI elements, etc.
+* [`ImageDrawable`]
 * Color types - see below.
-* Geometry - `Point`, `Size` and `Rectangle` provide ways of defining positions, dimensions and rectangular areas respectively.
+* Geometry - [`Point`], [`Size`] and [`Rectangle`] provide ways of defining positions, dimensions and rectangular areas respectively.
 
 ## Colors
 
-The `pixelcolor` module provides various standard color types, from `BinaryColor` to
-`Rgb888`. See the `pixelcolor` module documentation for the complete list of color depths
+The [`pixelcolor`] module provides various standard color types, from [`BinaryColor`] to
+[`Rgb888`]. See the [`pixelcolor`] module documentation for the complete list of color depths
 and formats available.
 
 ## Display drivers
 
-See the `DrawTarget` documentation for examples on how to integrate embedded-graphics with a
-display driver using the `DrawTarget` trait.
+See the [`DrawTarget`] documentation for examples on how to integrate embedded-graphics with a
+display driver using the [`DrawTarget`] trait.
 
 ## Images
 
-The `ImageDrawable` trait should be implemented for any image or image-like item, for example
+The [`ImageDrawable`] trait should be implemented for any image or image-like item, for example
 a spritemap.
 
 [embedded-graphics]: https://docs.rs/embedded-graphics
+[`Pixel`]: https://docs.rs/embedded-graphics-core/latest/embedded_graphics_core/drawable/struct.Pixel.html
+[`Point`]:  ./geometry/struct.Point.html
+[`Size`]:  ./geometry/struct.Size.html
```

Comment:
These should be modified too right?

Reply:
Good catch. The regex wasn't accepting more than one space in front of the link. I've made the regex less strict and updated `core/lib.rs`.